subtitles/whisper_transcriber.py

```diff
@@ -1,28 +1,34 @@
 from transcriber import Transcriber
 import whisper
 
-SRT_TIMESTAMP_FORMAT = "%02d:%02d:%06.3f"
+TIMESTAMP_FORMAT = "%02d:%02d:%06.3f"
 
 
 class WhisperTranscriber(Transcriber):
-    def __init__(self, model: str) -> None:
+    def __init__(self, model: str, debug: bool) -> None:
         """Initialize WhisperTranscriber with an array of given models.
 
         Args:
             model: The model type.
                 Visit https://github.com/openai/whisper for a list of available models.
+            debug: Display debug information or not.
         """
         super().__init__()
-        self.__model = whisper.load_model(model)
+        self.__model = model
+        self.__verbose = (None, True)[debug]
 
     def generate(self, source: str, language: str = None) -> (str, str):
-        # solve silence issue. see: https://github.com/openai/whisper/discussions/29
-        options = whisper.DecodingOptions(language=language).__dict__.copy()
-        options['no_speech_threshold'] = 0.275
-        options['logprob_threshold'] = None
+        options = whisper.DecodingOptions(language=language)
+        transcriber = whisper.load_model(self.__model)
```

Comment:
Everything should work now. Nevertheless, we probably should test how long `load_model` takes loading large language models since with the current implementation each request loads the model. RAM could potentially also be a bottleneck.